pkgs/by-name/sy/systemctl-tui/package.nix

```diff
@@ -1,30 +1,38 @@
-{ lib
-, rustPlatform
-, fetchCrate
-, stdenv
-, darwin
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+  nix-update-script,
+  testers,
+  systemctl-tui,
```

Comment:
This is crazy to me, but I'm seeing it in other packages. Can `finalAttrs` be used to remove this?

Reply:
`finalAttrs` could maybe work... I have not seen it used for `testVersion` but it could be possible.

Reply:
This is how it's done even when using finalattrs ...

Reply:
My mistake, I think we can use `finalAttrs.finalPackage` when using `finalAttrs`.

Reply:
It's not a big deal. I was just trying to better understand. Thanks!